.github/workflows/maven-publish.yml

```diff
@@ -32,7 +32,7 @@ jobs:
           server-password: MAVEN_PASSWORD
           gpg-passphrase: PASSPHRASE
       - name: Make a snapshot
-        run: mvn -Pdeploy -Pproduction --no-transfer-progress --batch-mode clean deploy -U
+        run: mvn --no-transfer-progress --batch-mode clean deploy -U
```

Comment:
the profiles should stay (-Pdeploy and -Pproduction)

Reply:
Ok!